src/tls/cockpit-certificate-ensure.c

```diff
@@ -408,7 +408,7 @@ main (int argc, char **argv)
   else if (argc == 2 && strcmp (argv[1], "--for-cockpit-tls") == 0)
     for_cockpit_tls = true;
   else
-    errx (EXIT_FAILURE, "usage: %s [--check]", argv[0]);
+    errx (EXIT_FAILURE, "usage: %s [--check | --for-cockpit-tls]", argv[0]);
```

Comment:
This was deliberately not documented, see commit 644116a0cdae. It's not something a user should *ever* use.

Do you actually need this for anything, or did it just look like a random omission?

Reply:
Oh, I thought it was an omission. I'll just drop the commit.